src/Finder/FinderRegistry.php

```diff
@@ -25,6 +25,9 @@ public static function getAllVariants(): array
             new DummyCpuCoreFinder(1),
             new HwLogicalFinder(),
             new HwPhysicalFinder(),
+            new OnlyOnLinuxFinder(
+                new LscpuFinder(),
+            ),
             new _NProcessorFinder(),
             new NProcessorFinder(),
             new NProcFinder(true),
```

Comment:
From the `physical` I think it means you can also add it to the `::getDefaultLogicalFinders()` and `::getDefaultPhysicalFinders()` lists

Reply:
Added